src/Extensions/ElementalPageExtension.php

```diff
@@ -52,7 +59,7 @@ public function getElementsForSearch()
             // CMS layout can break on the response. (SilverStripe 4.1.1)
             SSViewer::set_themes($oldThemes);
         }
-        return implode($output);
+        return implode(' ... ', $output);
```

Comment:
This will probably only need to be a space, we can't be adding dots into the content (not even an ellipsis).

Reply:
Only adding a single space will help to ensure words aren't immediately adjacent to one another, but it will also mean that sometimes searching for a phrase could result in incorrectly finding a phrase between the end of one block and the start of another...

Would you object to me making this configurable? I will then have it as a space by default, but developers can choose to use whatever delimiter they choose between blocks.

Reply:
Yep, get your point around phrase queries. This mechanism of concatenating block content isn't ideal, same as the rendered version. The best way would be to where the content is added into multi-value solr content field separately, not as a single value.
Configurable delimiter will do for now.